### ISSUE TITLE
Solve warning on ForstekFooter component

### ISSUE THREAD

**Issue by Angg** [forstek/timy-dashboard#34]
Line 6:3:  Emojis should be wrapped in <`span`>, have role="img", and have an accessible description with aria-label or aria-labelledby  jsx-a11y/accessible-emoji
